image/radon/radon.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.